src/modules/ce/components/Opportunity.tsx

```diff
@@ -10,7 +10,7 @@ import NotFound from '@/components/pages/NotFound';
 import useSafeParams from '@/hooks/useSafeParams';
 import ActivateReferralButton from '@/modules/ce/components/ActivateReferralButton';
 import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
-import { ClientDashboardRoutes, Routes } from '@/routes/routes';
+import { ClientDashboardRoutes, ProjectDashboardRoutes } from '@/routes/routes';
 import {
```

Comment:
Not sure where to put this comment: after denying a referral I see the opportunity status is "locked". I don't think that's quite right, my understanding is the opportunity should be immediately released into "open" state after a referral is fully denied, but we can discuss. 


<img width="197" alt="Screenshot 2025-03-14 at 11 53 51 AM" src="https://github.com/user-attachments/assets/6af3296a-24a5-4c73-a453-026c050d27ad" />

Reply:
Thanks! This is a backend issue, I've opened another PR https://github.com/greenriver/hmis-warehouse/pull/5210